pages/api/module/get.ts

```diff
@@ -0,0 +1,20 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { stringify } from "querystring";
+import { Module } from "../models/module";
+
+const get = async (
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> => {
+    const { id } = req.query;
+    const all = await Module.find({ _id: id.toString() });
```

Comment:
Does: `.findById(id).exec();` not work? I've seen it used in a couple other pr's and its working fine

Reply:
I'd prefer to keep things consistent across the codebase